APP: Move plugins to configurable folder (#479)
bccsa/luminary: app/README.md

> #### How to add plugins

Reviewer: ```suggestion
#### Adding plugins
```

-   To add a plugin you need to create a plugins folder, somewhere out of the project structure

> -   Set the VITE_PLUGIN_PATH env variable in your .env, then vite will go fetch your files at that location and copy it into the [plugins folder](./src/plugins/)

Reviewer: ```suggestion
-   Set the `VITE_PLUGIN_PATH` env variable in your .env for vite to copy your plugins to the [plugins folder](./src/plugins/)
```

```
VITE_PLUGIN_PATH="../../plugins"
```

> -   To let luminary run the plugin you need to add a env variable VITE_PLUGINS to your env file and add an array of the plugins you want to add

Reviewer: ```suggestion
-   To let luminary run the plugin you need to add an env variable `VITE_PLUGINS` to your env file with an array of plugin names.
```

-   Every plugin class should have a constructor function

> **The files is being copied everytime before vite build, dev, and test is run**

Reviewer: ```suggestion
**The files are copied before vite build, dev, and test runs**
```
